doc/man3/OSSL_PARAM.pod

```diff
@@ -131,6 +129,50 @@ The parameter data is a printable string.
 
 The parameter data is an arbitrary string of bytes.
 
+=item C<OSSL_PARAM_UTF8_PTR>
+
+The parameter data is a pointer to a printable string.
+
+The difference between this and C<OSSL_PARAM_UTF8_STRING> is that C<buffer>
+doesn't point directly at the data, but to a pointer that points to the data.
+
+This is used to indicate that constant data is or will be passed,
+and there is therefore no need to copy the data that is passed, just
+the pointer to it.
+
+C<buffer_size> will be set to the size of the data, not the size of
+the pointer to the data.
```

Comment:
I think I sometimes forget this...  would you mind telling me when I do?

Reply:
I'll try to remember :)